features/step_definitions/smokey_steps.rb

```diff
@@ -185,9 +185,12 @@ def should_see(text)
   expect(JSON.parse(@response.body).class).to eq(Hash)
 end
 
-When /^I see links to pages per topic$/ do
-  pages = Nokogiri::HTML.parse(page.body).css(".browse-container a")
-  unless pages.any?
+When /^I see the links pulled form search_api$/ do
+  latest_section = Nokogiri::HTML.parse(page.body).css("#latest")
+  latest_items_list = latest_section.css(".gem-c-document-list ul")
+  latest_items = latest_items_list.css(".gem-c-document-list__item-metadata")
+
+  unless latest_items.any?
```

Comment:
Shall we change the error here: `fail "There are no links on this Services and Information page"`

Reply:
I agree, this seems a bit vague and should tell us what we're expecting directly - if only because sometimes we have many of these failures and need a quick reference.